modules/indexer/issues/meilisearch/meilisearch.go

```diff
@@ -234,6 +234,7 @@ func (b *Indexer) Search(ctx context.Context, options *internal.SearchOptions) (
 	}
 
 	return &internal.SearchResult{
+		// TODO: if nonFuzzyWorkaround is not needed anymore this should deliver correct counts again
```

Comment:
unrelated nit ... but good to have it documented

once merged, I'll create an issue so others do not open new one and we have it tracked e.g. will link it to upstream too ...

Reply:
#29740 fix it :)